beets/autotag/match.py

```diff
@@ -18,7 +18,7 @@
 from __future__ import division
 
 import datetime
-import logging
+from beets import logging
```

Comment:
I would probably move this down next to the other (non-`__future__`) `from spam import eggs` declaration below. The same for other places where it's been changed in the middle of a sequence of `import`s.